### PR TITLE
Create Gno Studio intro.md

### PR DESCRIPTION
**Pull request by michelleellen**
This blog post announces Gno Studio and Gno Studio Connect

### CODE REVIEW

**Comment by michelleellen** [comment]
Image [
Twitter post - design - 10
](url)

**Comment by michelleellen** [comment]
@alexiscolin since the original is on Medium we will also need that the meta data refers to that in this version. Thank you!

**Comment by leohhhn** [comment]
Please let me know what tags you want for the blog post, who do you want the author of the blog post to be (displayed in the footer of the posts), and which date we want to publish this exactly. @michelleellen @ilgooz 

@alexiscolin - the metadata can currently only be what is in the frontmatter of other blog posts. Adding more stuff would require making modifications to the blog package. Maybe we can do this later.

**Comment by michelleellen** [comment]
Hi @leohhhn 

Blog, post, Gno Studio, dev, devx potential tags for the post. Can you actually add more images if possible https://medium.com/@gnostudio/introducing-gno-studio-the-premier-builder-suite-for-gno-land-d1b4d82b46de 

@alexiscolin how critical is the metadata given the duplicate content issue? 

Would publishing this week be possible? cc: @ilgooz

**Comment by alexiscolin** [comment]
@michelleellen I already checked with Leon and for now, it seems it's not possible to change the metadata for only one page. As we want medium to be the canonical page, we will have duplicate content issue. I don't think this is very critical as it's the first time it happens, but it will not help the SEO. For now, we can minimise the effect and improve the link juice by avoiding sharing this blog article (and sticking with the canonical version we chose in our communication on social media). Anyway, search engines will promote the one that is mostly shared and visited.

File: posts/2024-05-22_gno-studio-intro/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+publication_date: 2024-05-27T13:37:00Z
+slug: gno-studio-intro
+tags: [gno-studio, dev, devx, connect, gno.land]
```
Comment (ilgooz) [inline posts/2024-05-22_gno-studio-intro/README.md:4]: Can we change these with the following ones?
image

Comment (leohhhn) [reply posts/2024-05-22_gno-studio-intro/README.md]: [d74b73d](https://github.com/gnolang/blog/pull/84/commits/d74b73dc494c183a79210e5b377575643b4c2ecc)

**Pull request merged**